src/Integration.UnitTests/Connection/ConnectionWorkflowTests.cs

```diff
@@ -423,7 +259,6 @@ public async Task ConnectionWorkflow_ConnectionStep_UnsuccessfulConnection()
                 c.Should().Be(connectionInfo, "Unexpected connection");
                 p.Should().BeNull("Not expecting any projects");
             };
-            this.projectSystemHelper.Projects = new[] { new ProjectMock("foo.csproj") { ProjectKind = ProjectSystemHelper.CSharpProjectKind } };
```

Comment:
The project system helper isn't required any more, so I cleaned up those bits from the tests.